src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private static final int DIVIDER = 2;
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstLeg + secondLeg) / DIVIDER;
```

Comment:
The formula for the area of a right triangle is incorrect. It should be (firstLeg * secondLeg) / 2. Please correct the formula.